src/components/Card.js

```diff
@@ -21,11 +21,9 @@ class Card {
   }
 
   _setEventListeners() {
-    this._cardElement
-      .querySelector(".card__like-button")
-      .addEventListener("click", () => {
-        this._handleLikeClick(this);
-      });
+    this._likeButton.addEventListener("click", () => {
+      this._handleLikeClick(this);
+    });
 
```

Comment:
```js

  _handleLikeIcon() {
    this._cardElement
      .querySelector(".card__like-button")
      .classList.toggle("card__like-button_active");
  }
```


In `_handleLikeIcon` you need to use `this._likeButton` as well


```js
  _handleLikeIcon() {
    this._likeButton.classList.toggle("card__like-button_active");
  }


```

Reply:
completed.